spec/presenters/publishing_page_presenter_spec.rb

```diff
@@ -34,13 +34,17 @@
 
     context 'submission warning presenters' do
       let(:presenters) do
-        [submission_pages_presenter, from_address_presenter]
+        [submission_pages_presenter,
+         autocomplete_warning,
+         from_address_presenter]
```

Comment:
```suggestion
        [
          submission_pages_presenter,
          autocomplete_warning,
          from_address_presenter
        ]
```

Small formatting suggestion